databricks_blueprints/helpers.py

```diff
@@ -0,0 +1,63 @@
+import requests
+import shipyard_utils as shipyard
+
+
+# Create Artifacts folders
+base_folder_name = shipyard.logs.determine_base_artifact_folder(
+    'databricks')
+artifact_subfolder_paths = shipyard.logs.determine_artifact_subfolders(
+    base_folder_name)
+shipyard.logs.create_artifacts_folders(artifact_subfolder_paths)
+
+
+class DatabricksRequest(object):
```

Comment:
Pep formatting for the class is off, I think, there should be 2 empty lines between methods. Let me know if I'm wrong. 

Let's update the name of the class to `DatabricksClient` or `DatabricksRequests`. The name `DatabricksRequest` makes it seem like an instance is a single request to the databricks, whereas this class can be used multiple times for multiple requests. 

Also, below in main, let's update the name `db_request` to `client` or `databricks` or `databricks_client`. The name `db_request` could easily be interpreted as "database request" - it's unfortunate that databricks and database have the same db abbreviation, so I'd like to avoid the db abbreviation while working with databricks to avoid confusion.

Reply:
i'm pretty sure methods within a class are separated by one empty line.

As for the the databricks naming conventions, duly noted. The naming is definitely a bit off.